<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirmar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">self.setStyleSheet(&quot;background-color: white;&quot;)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/Confirmar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgb(255, 255, 255);
    color: #333333;
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border-radius: 5px;
}
</string>
    </property>
    <property name="text">
     <string>¿Estás seguro de que quieres eliminar?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>60</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_aceptar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    border-radius: 5px;
    border: 1px solid #CCC;
    color: #333;
    font-weight: bold;
    font-size: 12px;
    padding: 5px 20px;
}

QPushButton:hover {
    background-color: #E0F2F7;
    border: 1px solid #00AEEF;
    color: #333;
}

QPushButton:pressed {
    background-color: #00AEEF;
    border: 1px solid #00AEEF;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_cancelar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    border-radius: 5px;
    border: 1px solid #CCC;
    color: #333;
    font-weight: bold;
    font-size: 12px;
    padding: 5px 20px;
}

QPushButton:hover {
    background-color: #E0F2F7;
    border: 1px solid #00AEEF;
    color: #333;
}

QPushButton:pressed {
    background-color: #00AEEF;
    border: 1px solid #00AEEF;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>Button_aceptar</zorder>
   <zorder>Button_cancelar</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
